Builder/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_subdirectory(src)
+
+add_executable(Builder "builder.cpp")
+
+target_link_libraries(Builder PUBLIC CodeBuilder)
```

Comment:
Why are the includes missing from the CMakelIsts?

Reply:
They're not, they are included in CodeBuilder in src/CMakeLists.txt.